apps/leaguechamps/league_champs.star

```diff
@@ -0,0 +1,147 @@
+"""
+Applet: League Champs
+Summary: Display league characters
+Description: Shows league of legends champsions and their subtitle.
+Author: xl0lli
+"""
+
+load("cache.star", "cache")
+load("encoding/base64.star", "base64")
+load("encoding/csv.star", "csv")
+load("http.star", "http")
+load("random.star", "random")
+load("render.star", "render")
+load("schema.star", "schema")
+
+CACHE_TTL = 604800
+CSV_ENDPOINT = "https://gist.githubusercontent.com/xl0lli/bc6755ee77e52a9dcd481e75c95c74b6/raw/ac2c83cc874b2dbbe8f9449a8cef1df4ff8bedb4/league_champ_data"
```

Comment:
I noticed this CSV file is about 3.7M. The largest base64 encoded block is a 120x120 PNG file. You can probably reduce the size of the CSV by a lot if you pre-scale the image data down to the 24x32 that the app is rendering them at.

Reply:
compelted pre-scaling on file